packages/@uppy/status-bar/src/StatusBar.tsx

```diff
@@ -1,5 +1,6 @@
 import type { Body, Meta, UppyFile } from '@uppy/utils/lib/UppyFile'
 import type { Uppy, State } from '@uppy/core/src/Uppy.ts'
+import type { DefinePluginOpts } from '@uppy/core/lib/BasePlugin.ts'
```

Comment:
How about exporting all types we want to export on `@uppy/core`? I think that's less confusing and more clean for implementers of plugins.

Reply:
IMO `DefinePluginOpts` only makes sense for plugin implementer, so it makes sense to keep it in `BasePlugin` (we could also export it from `UIPlugin`) and we could keep `@uppy/core` types for end users only.